Fix errata

ansley opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#49903 Fix errata**

Differential Revision: [D25718411](https://our.internmc.facebook.com/intern/diff/D25718411)

Commit:
Fix errata

[ghstack-poisoned]

Commit:
Update on "Fix errata"


Differential Revision: [D25718411](https://our.internmc.facebook.com/intern/diff/D25718411)

[ghstack-poisoned]

SplitInfinity commented on torch/csrc/jit/mobile/export_data.cpp:
```diff
@@ -36,7 +36,7 @@ class ScriptModuleSerializer {
     std::vector<char> data;
     // Vector to capture the run-time class types during pickling the IValues
-    std::vector<c10::ClassTypePtr> memorizedClassTypes;
+    std::vector<c10::ClassTypePtr> memoizedClassTypes;
```
I think this change should be made inside `Pickler` too.

Commit:
Update on "Fix errata"


Differential Revision: [D25718411](https://our.internmc.facebook.com/intern/diff/D25718411)

[ghstack-poisoned]

Commit:
Update on "Fix errata"


Differential Revision: [D25718411](https://our.internmc.facebook.com/intern/diff/D25718411)

[ghstack-poisoned]
